### DISCUSSION TITLE
Galaxy Brain Badge Question

### DISCUSSION THREAD

**Discussion by dsp4302** [Schweinepriester/github-profile-achievements#39] [Q&A] [answered]
Apparently you need to get an accepted answer in discussion taking place in a public repository. What does it mean by public repository, and do I have all the criteria down?

**Comment by justin-2028** [comment, marked as answer]
Here is the criteria to achieve the 'Galaxy Brain' achievement:

- Answer at least 2 discussions
- Take note, you CANNOT mark your own answers to your own questions. They have to be marked by another user
- Discussions should be in a public repository (like this one)
- There's a current bug observed by some users whereby marking your own answers can prevent you from getting the badge instead

Check out this link for more info: https://github.com/orgs/github-community/discussions/18467

good luck on getting it

**Reply by dsp4302** [reply]
oh cool

**Reply by dsp4302** [reply]
just trying to get a badge to seem more knowledgeable than I am lol

**Reply by justin-2028** [reply]
you joined < 30 minutes ago, you should take a look around what GitHub has to offer first
